packetery/src/Packetery/Module/Plugin.php

```diff
@@ -695,4 +695,31 @@ public static function hideSubmenuItem( string $itemSlug ): void {
 	public static function getAppIdentity(): string {
 		return 'Woocommerce: ' . get_bloginfo( 'version' ) . ', WordPress: ' . WC_VERSION . ', plugin Packeta: ' . self::VERSION;
 	}
+
+	/**
+	 * Simplifies float value to have max decimal places.
+	 *
+	 * @param float|null $value            Value.
+	 * @param int        $maxDecimalPlaces Max decimal places.
+	 *
+	 * @return float|null
+	 */
+	public static function simplifyFloat( ?float $value, int $maxDecimalPlaces ): ?float {
```

Comment:
Třída Plugin se mi zdá už dost dlouhá, co založit nějaké Tools?

Reply:
Core Helper může být?